simphony/testing/abc_check_mesh.py

```diff
@@ -384,11 +400,14 @@ def test_update_item_points(self):
         container = self.container
         uids = self._add_items(container)
         item = self.get_operation(container, uids[2])
+        point_uids = [
+            container.add_point(Point((1.0 * i, 1.0 * i, 1.0 * i)))
```

Comment:
@nathanfranklin that was a really good catch. I could not see the problem because, the related code was in my mayavi branch. It should be ok now